modules/post/linux/gather/checkvm.rb

```diff
@@ -108,6 +113,23 @@ def run
       end
     end
 
+    # Check system vendor
+    if !vm
+      sys_vendor = read_file('/sys/class/dmi/id/sys_vendor')
+      if sys_vendor
+        case sys_vendor.gsub("\n", ' ')
+        when /qemu/i
+          vm = 'Qemu'
```

Comment:
Should this be `Qemu/KVM`? 
```suggestion
          vm = 'Qemu/KVM'
```